Client/clientclient.py

```diff
@@ -0,0 +1,37 @@
+import socket
+
+
+class Client:
+    def __init__(self):
+        self.client = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+
+    def connect_to_server(self):
+        self.client.connect(('localhost', 9009))
```

Comment:
why host and port are hardcoded? put them into instance attributes

Reply:
fixed